README.md

```diff
@@ -633,6 +633,27 @@ log.error(f"An exception was thrown for value $value%.12f: $ex")
 
 Do not nest `map` more than twice. Use named values/functions instead.
 
+## Handling Date And Time
+
+It is strongly recommended to use time library (either `java.time` or Joda Time) instead of doing calculations with milliseconds. As one implication of this rule, using System.currentTimeMillis()` is strongly discouraged and should not be used for time operations.
```

Comment:
Kevin says: Please make a clear statement for when to use unix timestamp, and when to use date/time library. When serialize and store to DB, unix timestamp is the only choice with no exception. date/time library should only be used when cross-day time manipulation is required.